bumble/profiles/asha_service.py

```diff
@@ -60,12 +60,12 @@ def __init__(self, capability: int, hisyncid: List[int], device: Device, psm=0):
         self.psm = psm  # a non-zero psm is mainly for testing purpose
 
         # Handler for volume control
-        def on_volume_write(_connection, value):
+        def on_volume_write(connection, value):
             logger.info(f'--- VOLUME Write:{value[0]}')
-            self.emit('volume', value[0])
+            self.emit('volume', connection, value[0])
 
         # Handler for audio control commands
-        def on_audio_control_point_write(_connection, value):
+        def on_audio_control_point_write(connection, value):
```

Comment:
consider adding a type to connection

```suggestion
        def on_audio_control_point_write(connection: Connection, value):
```